src/main/java/com/fasterxml/jackson/databind/DeserializationFeature.java

```diff
@@ -272,6 +272,19 @@ public enum DeserializationFeature implements ConfigFeature
      */
     FAIL_ON_TRAILING_TOKENS(false),
 
+    /**
+     * Feature that determines behavior when deserializing polymorphic types of
+     * Id.CLASS or Id.MINIMAL_CLASS type if the subtype is not explicitly registered.
+     * For types like Id.NAME, you already need to declare the subtypes but with
+     * Id.CLASS and Id.MINIMAL_CLASS you do not. If enabled, an exception will be
+     * thrown if a subtype is encountered that has not been explicitly registered.
+     *<p>
+     * Feature is disabled by default.
+     *
+     * @since 2.19
+     */
+    FAIL_ON_POLYMORPHIC_SUBTYPE_CLASS_NOT_EXPLICITLY_REGISTERED(false),
```

Comment:
While I like explicit and descriptive names, we may want to shorten this a bit :)
(I know not a big deal)

Reply:
Naming ideas!

- FAIL_ON_UNREGISTERED_POLYMORPHIC_SUBTYES
- FAIL_ON_UNREGISTERED_SUBTYES

Reply:
Thanks! Name needs to include `CLASS` somewhere tho, as it only applies to that mode
(put another way: with Type Name as id this will always happen as registration is needed)

Reply:
FAIL_ON_SUBTYPE_CLASS_NOT_REGISTERED is my latest name for the feature - is this ok?

Reply:
That sounds good indeed.